content/components/core/code/scss-variables.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: section
+---
+
+## SCSS Variables
+
+### Default Theme Colours
+
+| Variable Name               | Default Value | Description                                                                                                |
+|-----------------------------|---------------|------------------------------------------------------------------------------------------------------------|
+| `$AU-color-foreground-text`   | `#313131`       | Colour used for paragraphs, headings etc.                                                                                 |
```

Comment:
Not sure if we need `etc.` feels a bit vague. I would leave it as `paragraphs and headings`